tasmota/support_features.ino

```diff
@@ -254,8 +254,8 @@ void GetFeatures(void)
 #ifdef USE_SHT
   feature_sns1 |= 0x00000100;  // xsns_07_sht1x.ino
 #endif
-#ifdef USE_HTU
-  feature_sns1 |= 0x00000200;  // xsns_08_htu21.ino
+#if defined(USE_HTU) || defined(USE_HDC1080)
```

Comment:
Don't do this. If both HTU and HDC are connected use the correct online commands to disable one of them (your code must support this. See HTU code what I mean).

Don't update support_features.ino any way. I'll dot the administration.

Reply:
Ok understood, I would be curious however as to why exactly we are using these flags, and the choice of having a driver being assigned a specific bit from this 4 byte word.

Reply:
The bits pop-up in status message ``status 4`` and tell all assisting users what features are enabled in the binary. Only for debugging and helping the ones in need (why does my driver not work). If you only knew how much debug info is in this code...

Reply:
I see. Regarding "the correct online commands", you mean the I2cDriverXX [0|1] commands? But aren't these using the driver index # to achieve this?

Reply:
Nope. The index in I2CDEVICES.md is used as i2c is used by sensors, drivers and displays all with their own xxxx_yy_name.ino file.